source/configuration/modules/pmrfc3164.rst

```diff
@@ -43,6 +43,16 @@ Parser Parameters
    format. The hostname field sent by syslog-ng may be prefixed by the source
    name followed by an at-sign character.
 
+.. function:: force.tagEndingByColon <boolean>
+
+   **Default**: off
+
+   Available since: 8.24.0
+
+   This setting tells the parser that tag need to be ending by colon to be valid.
+   In others case, the tag is the first word of the message and the tag is left
+   empty.
+
 .. function:: detect.YearAfterTimestamp <boolean>
```

Comment:
Please explain what happens to the syslogtag property if the tag does not end in colon.